atomic-swap/.gitignore

```diff
@@ -0,0 +1 @@
+/build
```

Comment:
This is something for a global gitignore.

Reply:
All the Gradle modules have the same .gitignore file, but since the global one already covers it, I guess ours can be safely removed.